### PR TITLE
Added a mnemonic for the 4 a11y principles

### PR DESCRIPTION
**Pull request by hashkitti001**
## Because
It provides a mnemonic for the four accessibility principles which would aid in remembering the concepts

## This PR
Addition of text

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [ ] The `Because` section summarizes the reason for this PR
-   [ ] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ It's important to understand that while the WCAG can be incredibly helpful for b
 ### The Four Principles
 
 The WCAG are organized around four, core principles that should be kept in mind when implementing any sort of accessibility feature:
```
Comment (KevinMulhern) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:20]: Could we fit the acronym in more naturally?

Maybe with something like this:
```suggestion
The WCAG are organized around four, core principles (POUR) that should be kept in mind when implementing any sort of accessibility feature:
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md (hunk 1 above)
Comment (hashkitti001) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:20]: Great idea. How can I change my pull request 😅.Kinda new to GitHub myself

Comment (KevinMulhern) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:20]: No worries, you can edit the file directly in the file [changed tab](https://github.com/TheOdinProject/curriculum/pull/25368/files)

Screenshot 2023-04-19 at 11 10 13

**Review by KevinMulhern — APPROVED** [review]
Great stuff, thanks @hashkitti001

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ It's important to understand that while the WCAG can be incredibly helpful for b
 
 ### The Four Principles
 
-The WCAG are organized around four, core principles that should be kept in mind when implementing any sort of accessibility feature:
+The WCAG are organized around four, core principles (POUR) that should be kept in mind when implementing any sort of accessibility feature:
+You can use the acronym POUR for this, which stands for:
```
Comment (KevinMulhern) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:21]: Sorry, I missed this. Now that we have it in the introductory text, can this be removed please?

```suggestion
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

**Review by KevinMulhern — APPROVED** [review]

**Pull request merged**